### PR TITLE
Debt/met 5966 align harvesting methods

### PR DESCRIPTION
**Pull request by jochen-vermeulen**

### CODE REVIEW

File: metis-harvesting/src/main/java/eu/europeana/metis/harvesting/http/HttpHarvesterImpl.java [hunk 1]
```diff
@@ -258,24 +331,38 @@ public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs) {
...
+    @Override
+    public ByteArrayInputStream getContent() {
+      return entryContent;
     }
 
     @Override
-    public String getEntryName() {
-      return entryName;
+    public boolean isDeleted() {
+      return false;
```
Comment (JoanaCMS) [inline metis-harvesting/src/main/java/eu/europeana/metis/harvesting/http/HttpHarvesterImpl.java:354]: It always returns false, is this intentional?

File: metis-harvesting/src/main/java/eu/europeana/metis/harvesting/http/HttpHarvesterImpl.java [hunk 2]
```diff
@@ -258,24 +331,38 @@ public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs) {
...
     @Override
+    @Deprecated
     public ByteArrayInputStream getEntryContent() {
-      return entryContent;
+      return getContent();
+    }
+
+    @Override
+    public Instant getTimeStamp() {
+      return null;
```
Comment (JoanaCMS) [inline metis-harvesting/src/main/java/eu/europeana/metis/harvesting/http/HttpHarvesterImpl.java:365]: Also here, it always returns null

File: metis-harvesting/src/main/java/eu/europeana/metis/harvesting/http/HttpHarvesterImpl.java (hunk 1 above)
Comment (jochen-vermeulen) [reply metis-harvesting/src/main/java/eu/europeana/metis/harvesting/http/HttpHarvesterImpl.java:354]: Yes. Other protocols (such as OAI) sometimes return records or record IDs that are deleted at source (with a flag saying that it is deleted). However, this is not the case here. In a zip file we can't make that distinction (we don't have such a flag) and we assume that all records are active and to be harvested.

File: metis-harvesting/src/main/java/eu/europeana/metis/harvesting/http/HttpHarvesterImpl.java (hunk 2 above)
Comment (jochen-vermeulen) [reply metis-harvesting/src/main/java/eu/europeana/metis/harvesting/http/HttpHarvesterImpl.java:365]: Similar story. Other protocols (can) maintain a timestamp for a record (i.e. last change time). In a zip file we don't have this.

**Pull request merged**